lnrpc/lightning.proto

```diff
@@ -3357,6 +3361,62 @@ message RouteHint {
     repeated HopHint hop_hints = 1;
 }
 
+message BlindedPayment {
+    // The blinded route to send the payment to.
+    BlindedRoute route = 1;
+
+    // The relay parameters for the blinded route.
+    BlindedRelay relay_parameters = 2;
+
+    // The routing constraints for the blinded route.
+    BlindedConstraints relay_constraints = 3;
+
+    // The feature bits for the route.
+    repeated FeatureBit features = 4;
+}
+
+message BlindedRelay {
+    // The aggregate base fee for hops in the blinded portion of the route,
+    // expressed in millisatoshis.
+    uint64 aggregate_base_fee_msat = 1;
+
+    // The aggregate proportional fee for hops in the blinded portion of the
+    // route, expressed in micro-satoshis (parts per million).
+    uint64 aggregate_proportional_fee_ppm = 2;
+
+    // The total cltv expiry delta for the blinded portion of the route.
+    uint32 total_cltv_delta = 3;
+}
+
+message BlindedConstraints {
+    // The minimum hltc size for the route.
+    uint64 min_htlc = 1;
+
+    // The cltv limit for the route.
+    uint32 cltv_limit = 2;
+}
+
+message BlindedRoute {
+    // The unblinded pubkey of the introduction node for the route.
+    bytes introduction_node = 1;
+
+    // The ephemeral pubkey used by nodes in the blinded route.
```

Comment:
Q: what if we combine a receiver and sender blinded route, do these fields still need to be set?